_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,149 @@
+      1. having a business relationship (such as serving as a consultant for or part owner of same company) within 48 months.
+
+   1. No associate editor will assign a reviewer who is known to have a conflict of interest with any author.  The explanation of conflict of interest will be explained to the reviewer in their invitation, and the reviewer will affirm that they do not have a conflict of interest.
+
+   1. If later it is discovered that a reviewer has a conflict of interest with an author during the review process, this review will be discarded, and another review will be sought.
+
+   1. If a conflict of interest in a reviewer is found after acceptance, the associate editor will evaluate whether any bias may have affected the acceptance. If the associate editor feels they may be bias, a committee of three associate editors selected by the Section Lead Editor will investigate whether any bias or fraud was intended, and will follow procedures to determine whether admission was fraudulent as outlined by the bylaws on fraud.
+
+   1. No Section Lead Editor shall assign an Associate Editor if they
+have a conflict of interest with a paper. No Associate Editor may
+manage a paper if they have a conflict of interest, and shall notify
+the Section Lead Editor if they do have a conflict.
+
+   1. If a Section Lead Editor has a conflict of interest, the
+assignments to Associate Editors will be made by another Section Lead
+Editor.  The order will be cyclic; if Lead Section Editor A has a
+conflict, it will got to Lead Section Editor B, unless they also have
+a conflict, in which case it will go to C, and so on, returning to A
+when the sections are exhausted. In the case that all of the lead
+editors have conflicts, the Managing Editors will jointly pick an
+Associate Editor who will then assign another Associate Editor to
+manage the manuscript.
+      1. Comparison of Molecular Simulation Package Editor
+      1. Tutorials Editor
+      1. Best Practices Editor
+      1. Review Editor
+      1. Lessons Learned Editor
+      1. continuing with editors of other sections, in order they are chronologically added to the journal.
+
+1. Fraud and Plagiarism
+
+    1. Definitions:
+
+       1. 'Plagiarism' is defined as including any clearly recognizable
+amount of text from a previous publication without attribution.
+
+       1. 'Fraud' is defined as any false representation of the circumstances of the paper or the data contained in the paper. It could include fraudulent data, or fraudulent reviews.
+
+    1. If there is a suspicious or accusation of fraud or plagiarism, a
+       committee of three members of the editorial board will be chosen by
+       the managing editors to investigate.  If this committee votes
+       unanimously that fraud or plagiarism occurred, then the paper will
+       be removed from the journal, and the author will not be allowed to
+       submit to LiveCoMS for three years.
```

Comment:
Do we want to note that, say, in the case of findings of serious misconduct we might also report it to appropriate authorities at the authors' institution(s)?

Reply:
If we do, what is the confidentiality level of our internal info and proceedings?

Reply:
I think we could do this without passing on internal information. Specifically, if someone submits a manuscript which steals from someone else or commits plagiarism, then we can simply give the appropriate entities a copy of the submitted work and raise our concerns in summary and they would investigate. (They would investigate anyway even if we shared our internal info so I don't see it as helping that much.)

I don't expect to exercise this, but in imagining the scenario where someone submits a plagiarized paper, I would be pretty bummed if the only thing we could do about it was tell them that they can't submit to LiveCoMS for three years. In my mind it should (as in the case of misconduct in normal journals, etc.) potentially trigger a "report the problem to their supervisor/institution" thing. A ban from submitting to us for three years isn't going to be much of a consequence for anyone.